packages/wrangler/src/pipelines/validate.ts

```diff
@@ -0,0 +1,36 @@
+export function validateName(label: string, name: string) {
+	if (!name.match(/^[a-zA-Z0-9-]+$/)) {
+		throw new Error(`Must provide a valid ${label}`);
```

Comment:
It would be better to fail these check with an UserError. We used it to differentiate whether it is a possible bug on our side or if it is simply a user error.